app/models/spree/sale_price.rb

```diff
@@ -1,9 +1,18 @@
 module Spree
   class SalePrice < ActiveRecord::Base
-    include Spree::SoftDeletable
+    if self.respond_to?(:acts_as_paranoid)
```

Comment:
Won't this be true within any application that has paranoia loaded, regardless of the soft deletion method available in Solidus?

Reply:
Has been removed in favor of Solidus version to be more explicit. Please let me know if you have any other ideas in which to based this conditional off of.